server/middleware/authorization.js

```diff
@@ -0,0 +1,33 @@
+const jwt = require('jsonwebtoken');
+const config = require('../../config/config');
+const { AuthModel } = require('../../server/auth/auth.model');
+const redis = require('../../index.redis');
+const asyncHandler = require('./async');
+const ErrorResponse = require('../utils/errorResponse');
+
+exports.verifyToken = asyncHandler(async (req, res, next) => {
+  const bearerHeader = req.headers.authorization;
+  if (bearerHeader && bearerHeader.startsWith('Bearer')) {
+    const accessToken = bearerHeader.split(' ')[1];
+
+    if (!accessToken) return next(new ErrorResponse('Not authorized !!', 401));
+
+    try {
+      const isBlackListed = await redis.get(`BlackListed${accessToken}`);
+      if (isBlackListed) {
+        return next(new ErrorResponse('This Token is already used', 403));
```

Comment:
You should say 'the token is invalid' and check the status code.. Is it `403` or `401`

Reply:
Thank you. Changed.